app/collins/util/LshwRepresentation.scala

```diff
@@ -105,6 +113,7 @@ case class LshwRepresentation(
         (cpuCoreCount == other.cpuCoreCount) &&
         (cpuThreadCount == other.cpuThreadCount) &&
         (cpuSpeed == other.cpuSpeed) &&
+        (gpuCount == other.gpuCount) &&
```

Comment:
i am not sure where this method gets used, but comparing GPU counts (as opposed to deep comparison of desc/vendor/product) could produce odd behavior when you change GPUs and update LSHW, but the same number of GPUs are present. Would collins not update the meta attributes in the DB if the old reconstructed LSHW `equals` the newly submitted one?

I think what you have in this diff is fine, but we should test this behavior (induct with GPU A, change to GPU B, induct, ensure LSHW is updated properly) and open an issue if this is a problem